src/main/java/org/opentripplanner/middleware/models/TripMonitorAlertNotification.java

```diff
@@ -53,39 +56,64 @@ public static TripMonitorAlertNotification createAlertNotification(
         // If there are any unseen alerts, include list of these.
         if (!unseenAlerts.isEmpty()) {
             newAlertsNotification = new TripMonitorAlertSubNotification(
-                "New alerts found:",
-                unseenAlerts
+                unseenAlerts,
+                String.format("%s found:", formatAlertCount(unseenAlerts, Set.of())),
+                NEW_ALERT_ICON
             );
         }
         // If there are any resolved alerts, include list of these.
+        boolean isAllClear = false;
         if (!resolvedAlerts.isEmpty()) {
+            isAllClear = previousAlerts.size() == resolvedAlerts.size() && unseenAlerts.isEmpty();
             resolvedAlertsNotification = new TripMonitorAlertSubNotification(
                 // If all previous alerts were resolved and there are no unseen alerts, send ALL CLEAR.
-                previousAlerts.size() == resolvedAlerts.size() && unseenAlerts.isEmpty()
+                resolvedAlerts,
+                isAllClear
                     ? "All clear! The following alerts on your itinerary were all resolved:"
-                    : "Resolved alerts:",
-                resolvedAlerts
+                    : String.format("%s:", formatAlertCount(Set.of(), resolvedAlerts)),
+                RESOLVED_ALERT_ICON
             );
         }
-
-        return new TripMonitorAlertNotification(newAlertsNotification, resolvedAlertsNotification);
+        String summary = getSummary(unseenAlerts, resolvedAlerts, isAllClear);
+        return new TripMonitorAlertNotification(newAlertsNotification, resolvedAlertsNotification, summary);
     }
 
-    /**
-     * Basic text formatting of the alert notification.
-     */
-    public static String getBody(
-        TripMonitorAlertSubNotification newAlertsNotification,
-        TripMonitorAlertSubNotification resolvedAlertsNotification
+    public static String getSummary(
+        Set<LocalizedAlert> newAlerts,
+        Set<LocalizedAlert> resolvedAlerts,
+        boolean isAllClear
     ) {
         StringBuilder body = new StringBuilder();
-        if (newAlertsNotification != null) {
-            body.append(newAlertsNotification);
-            body.append(System.lineSeparator());
-        }
-        if (resolvedAlertsNotification != null) {
-            body.append(resolvedAlertsNotification.toString(true));
+        // If there are any unseen or resolved alerts, notify accordingly.
+        boolean hasNewAlerts = !newAlerts.isEmpty();
+        boolean hasResolvedAlerts = !resolvedAlerts.isEmpty();
+        if (isAllClear) {
+            body.append("☑ All clear! All alerts on your itinerary were all resolved.");
```

Comment:
Replace icon with `RESOLVED_ALERT_ICON`

Reply:
Refactored in https://github.com/ibi-group/otp-middleware/pull/205/commits/1d8679e83d0b7b20c6fa157d54bea004d9f10b67.